### PR TITLE
Standardize Eslint and Prettier

### PR DESCRIPTION
**Pull request by JCSanPedro**
Apply eslint/prettier standards that we have in other sylo projects, such as https://github.com/dn3010/glorious-mobile.

This PR's changes doesn't need much careful review. Basically just copied the same config files from other projects, then ran

`eslint */*.ts --fix` and
`prettier */*.ts --write`

### CODE REVIEW

**Review by JoshD641 — APPROVED** [review]

**Review by teinnt — COMMENTED** [review]
Yes, the PR looks right to me. Can you also add this file https://github.com/dn3010/glorious-mobile/blob/main/.prettierrc.js and run the `npx prettier '{*/,}*.{js,ts,json,*.js,*.ts}' --write && npx eslint '*/*.{js,ts,json,*.js,*.ts}' --fix` again?

File: .eslintignore [hunk 1]
```diff
@@ -2,3 +2,7 @@ node_modules
 artifacts
 cache
 coverage
+typechain
+abi
+scripts
+*/**/*.json
```
Comment (teinnt) [inline .eslintignore:8]: Can you add the last line here?

**Review by teinnt — COMMENTED** [review]
Just one more very small thing!

File: .eslintignore (hunk 1 above)
Comment (JCSanPedro) [reply .eslintignore:8]: Good catch!

**Review by teinnt — APPROVED** [review]

**Review by JoshD641 — APPROVED** [review]

**Pull request merged**